app/models/recipe.rb

```diff
@@ -27,4 +27,15 @@ def remaining_capacity
   validates_presence_of :grains,  message: "must include name and weight of at least one grain"
   validates_presence_of :hops,  message: "must include name and weight of at least one hop"
   
+
+  def original_gravity
```

Comment:
It'd be good to have some test coverage for `original_gravity` and `total_grain_weight` in the recipe model spec.